src/long_leaf_exports.jl

```diff
@@ -134,6 +135,7 @@ export
     GtkTextMarkLeaf,
     GtkTextTagLeaf,
     GtkTextViewLeaf,
+    GtkClipboardLeaf,
     GtkToolButtonLeaf,
     GtkToolItemLeaf,
     GtkToolbarLeaf,
```

Comment:
please make the corresponding updates to `short_leaf_exports.jl` and `short_exports.jl`